packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -288,7 +288,10 @@ export default function vercelServerless({
 				}
 			},
 			'astro:build:ssr': async ({ entryPoints, middlewareEntryPoint }) => {
-				_entryPoints = entryPoints;
+				const entryPointsWithoutPrerenderedRoutes = new Map(
+					Array.from(entryPoints).filter(([routeData]) => !routeData.prerender)
+				);
+				_entryPoints = entryPointsWithoutPrerenderedRoutes;
```

Comment:
```suggestion
				_entryPoints = new Map(
					Array.from(entryPoints).filter(([routeData]) => !routeData.prerender)
				);
```

Reply:
Thanks for the quick review, @ematipico! I've applied the suggested change 😊